arbitrum-docs/for-devs/dev-tools-and-resources/oracles.mdx

```diff
@@ -12,26 +11,10 @@ import KnowMoreToolsBox from '../partials/_know-more-tools-box-partial.md';
 
 The following [oracles](../../for-devs/concepts/oracles.mdx) will help you integrate off-chain data into your decentralized apps (dApps):
 
-## Chainlink
+| Name      | Description                                                                                                                                                                                                                                                                                  | Example/Documentation                                                       |
```

Comment:
A bit less sales language; more table

Reply:
To be completely honest, I'd remove the Oracle page that mention third-party solutions 😅 "How to use oracles in Arbitrum" . I think that's going to cause more trouble than other.
We could maybe remove that one and leave the reference page. But this might be a discussion for another moment (and another PR).

Reply:
@TucksonDev I've linked to this [here](https://www.notion.so/arbitrum/Dev-docs-V3-5-Orbit-Consolidation-vs-Expansion-d3525190888b4dea99fb378a3d744f45?pvs=4#c0c3eb32223d459589c3673464f861b5) so we don't forget about it 🙏